File: web/src/app/models/dto/usuario-logado-dto.ts
export interface UsuarioLogadoDTO {
  id: string | null;
  nomeCompleto: string | null;
  nomeLogin: string | null;
  email: string | null;
  token: string | null;
  perfis: string[];
  autenticado: boolean;
  origem: 'keycloak' | 'sistema';
}

File: api/Feak/Feak.SistemaFinanceiro.DomainService/DTOs/UsuarioLogadoDTO.cs
namespace DomainService.DTOs;

public class UsuarioLogadoDTO
{
    public string? Id { get; set; }
    public string? NomeLogin { get; set; }
    public string? NomeCompleto { get; set; }
    public string? Email { get; set; }
    public bool Autenticado { get; set; }
    public IEnumerable<string> Perfis { get; set; } = Enumerable.Empty<string>();
}